.gitignore

```
@@ -0,0 +1,3 @@
+cmd/go-tfidf/go-tfidf
+.gomodcachedir
+.devimgbuild.logs
```

we can ignore coverage files too

```suggestion
.devimgbuild.logs
coverage.html
coverage.out
```

---

@julianogalgaro Done!